Neonatal Sepsis model (#1569)
ihmeuw/vivarium_research: docs/source/models/causes/neonatal/neonatal_sepsis.rst

Assumptions and Limitations
+++++++++++++++++++++++++++

Focusing solely on YLLs will likely underestimate the total burden of sepsis and, consequently, the DALYs averted by interventions that reduce sepsis risk. However, we anticipate this underestimation to be less than 10%.

Our approach of dividing overall mortality risk into subcause-specific mortality risks is an approximation that may either overestimate or underestimate the burden of sepsis, depending on the other included causes and their order of consideration. Given that these mortality risks are relatively small, we expect this to result in an error of less than 1%.

Cause Model Diagram
+++++++++++++++++++

We are not modeling Neonatal Sepsis dynamically as a finite state machine, but we can draw an directed 
graph to represent the collapsed decision tree  
representing this cause. Unlike a state machine representation, the values on the 
transition arrows represent decision probabilities rather than rates per 
unit time.

.. graphviz::

    digraph NN_sepsis_decisions {
        rankdir = LR;
        lb [label="live birth", style=dashed]

>         nn_alive [label="neonate survived"]

Reviewer: ```suggestion
        nn_alive [label="neonate did not \ndie of sepsis"]
```

>         nn_dead [label="neonate died"]

Reviewer: ```suggestion
        nn_dead [label="neonate died \nof sepsis"]
```

        lb -> nn_alive  [label = "1 - mr"]
        lb -> nn_dead [label = "mr"]
    }

.. list-table:: State Definitions
    :widths: 7 20
    :header-rows: 1

    * - State
      - Definition
    * - live birth
      - The parent simulant has given birth to a live child simulant (which
        is determined in the
        intrapartum step of the :ref:`pregnancy model
        <other_models_pregnancy_closed_cohort_mncnh>`)

>     * - neonate survived

Reviewer: ```suggestion
    * - neonate did not die of sepsis
```

>       - The child simulant survived for the first 28 days of life

Reviewer: ```suggestion
      - The child simulant did not die of sepsis for the first 28 days of life
```

>     * - neonate died

Reviewer: ```suggestion
    * - neonate died of sepsis
```

>       - The child simulant died within the first 28 days of life

Reviewer: ```suggestion
      - The child simulant died of sepsis within the first 28 days of life
```
